src/rules/wcag20/levelA/touchable-has-alt.ts

```diff
@@ -31,21 +35,46 @@ export const rule: Rule.RuleModule = {
         )
       ) {
         if (!hasEveryProp(node.attributes, [ACCESSIBLE, ACCESSIBILITY_LABEL])) {
-          const hasAltElement = findChild(parent, (child) => {
+          // Checks if the child element has a label element.
+          const labelElement = findChild(parent, (child) => {
             return (
               hasEveryProp(child.attributes, [
                 ACCESSIBLE,
                 ACCESSIBILITY_LABEL,
               ]) || isTargetElement(child, context.options, [TEXT], TEXT)
             )
           })
-
-          if (!hasAltElement) {
+          if (!labelElement) {
             context.report({
               node,
               message:
                 'The `Touchable` Element must have accessible text. Need to use `accessible` prop and `accessiblityLabel` for the `Touchable` Element to make it accessible.',
             })
+            return
+          }
+        }
+        // If the only label was `Image`, then the `imagebutton` role is required.
+        if (context.options[0]?.__experimentalCheckRole === true) {
```

Comment:
Temporarily check for the `__experimentalCheckRole` option to pilot the `imagebutton` check.